src/sentry/models/eventattachment.py

```diff
@@ -92,7 +96,7 @@ class Meta:
 
     __repr__ = sane_repr("event_id", "name")
 
-    def delete(self, *args, **kwargs):
+    def delete(self, *args: Any, **kwargs: Any) -> tuple[int, dict[str, int]]:
```

Comment:
isn't object better for this?

Reply:
for parameters `object` wouldn't work because the underlying function it calls _is_ typed and so `object` wouldn't fit in a `bool` for instance